src/Handler/Locator/InMemoryLocator.php

```diff
@@ -15,7 +15,7 @@
  *      $inMemoryLocator->addHandler($myHandler, 'My\TaskAddedCommand');
  *
  *      // Returns $myHandler
- *      $inMemoryLocator->getHandlerForCommand(new My\TaskAddedCommand());
+ *      $inMemoryLocator->getHandlerForCommand('My\TaskAddedCommand');
```

Comment:
@mvriel @rosstuck 

Given the time that has passed, and given that PHP 5.5 is not supported anymore, maybe we can update this?

Also see #102.